### PR TITLE
KAZOO-5165 unlimited modb queries

### PR DESCRIPTION
**Pull request by jamhed**

### CODE REVIEW

File: applications/crossbar/src/crossbar_view.erl [hunk 1]
```diff
@@ -102,10 +95,10 @@ ascending_end_key(Context, StartKey) ->
 
 page_size() -> ?PAGINATION_PAGE_SIZE.
 page_size(Context) -> page_size(Context, cb_context:api_version(Context)).
-page_size(_Context, ?VERSION_1) -> 'undefined';
+page_size(_Context, ?VERSION_1) -> undefined;
```
Comment (lazedo) [inline applications/crossbar/src/crossbar_view.erl]: @jamhed please do not perform unrelated changes. lots of these in the pr

Comment (jamhed) [inline applications/crossbar/src/crossbar_view.erl]: @lazedo, I rewrote a lot of it, so take it all as new.

**Comment by fenollp** [comment]
Shouldn't this go to 4oh too?

**Comment by lazedo** [comment]
only bugs go into 4.0 this is a changing behaviour
